op-node/rollup/derive/batches_test.go

```diff
@@ -16,25 +18,38 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 )
 
-type ValidBatchTestCase struct {
+type ValidSingularBatchTestCase struct {
 	Name       string
 	L1Blocks   []eth.L1BlockRef
 	L2SafeHead eth.L2BlockRef
 	Batch      BatchWithL1InclusionBlock
 	Expected   BatchValidity
 }
 
+type ValidSpanBatchTestCase struct {
+	Name             string
+	L1Blocks         []eth.L1BlockRef
+	L2SafeHead       eth.L2BlockRef
+	OriginChangedBit uint
+	SingularBatches  []*SingularBatch
+	L1InclusionBlock eth.L1BlockRef
```

Comment:
Why maintain L1InclusionBlock, SingularBatches, TxData, OriginChangedBit instead of single span batch(type `Batch`)? Is it because too nested?